### ISSUE TITLE
pyautogui.PAUSE function doesn't work

### ISSUE THREAD

**Issue by baraksoa** [asweigart/pyautogui#480]
Hey,

pyautogui.PAUSE function doesn't work for me, for example in this code:

```
import pyautogui

pyautogui.PAUSE = 2.5

pyautogui.hotkey('enter')
pyautogui.hotkey('ctrl', 'enter')
pyautogui.hotkey('ctrl', 'z')
```

PyAutoGUI functions doesn't wait 2,5 seconds after performing actions, the program run takes 1 sec only. Something is missing or I'm misunderstand this function?

Windows 10, and I tried in python 3.7 and 3.8 too. Thanks a lot!

**Comment by asweigart** [comment]
Thanks for pointing this out! I've fixed it in 0.9.51

**Comment by asweigart** [comment]
Ah, something got switched around in the merge. It should be in 0.9.52 now.
